doc/Makefile

```diff
@@ -14,6 +14,10 @@ PYTHON = python$(PYVER)
 SPHINXOPTS    ?=
 SPHINXBUILD   ?= LANG=C sphinx-build
 PAPER         ?=
+# For merging a documentation archive into a git checkout of numpy/doc
+# Turn a tag like v1.18.0rc1 into 1.18
```

Comment:
`rc1` shouldn't include a doc publishing, so not needed

Reply:
oh, but from reading further down: you also want to change `v1.18.1` into `1.18`. so it's just the comment that needs updating

Reply:
ok